spec/controllers/calendars_controller_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe CalendarsController do
+  render_views
+
+  let!(:conference) { create(:conference, splashpage: create(:splashpage, public: true), venue: create(:venue)) }
+  let!(:events) { create_list(:event_scheduled, 2, program: conference.program) }
+
+  describe 'GET #index' do
+    before :each do
+      conference.program.update_attributes(schedule_public: true)
```

Comment:
Replacing this and the next occurance of `update_attributes` to `update` makes this a fully working and ci-passing PR